src/js/actions/tools.js

```diff
@@ -878,7 +879,8 @@ define(function (require, exports) {
                 specs.push({
                     key: utilShortcuts.GLOBAL.TOOLS.SHAPE,
                     modifiers: {},
-                    fn: activateTool
+                    fn: activateTool,
+                    name: "Other " + tool.name.toLowerCase() + " shortcut"
```

Comment:
This is a hidden shortcut and is duplicate with Rectangle tool. I am wondering if we should just hide it from the search as well.